AdminWebsite/AdminWebsite/ClientApp/src/app/booking/endpoints/endpoints.component.html

```diff
@@ -19,9 +19,6 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                     <li *ngIf="duplicateDa">
                         Please enter a unique display name
                     </li>
-                    <li *ngIf="duplicateDefAdv">
-                        Please link a unique defence advocate to an endpoint
-                    </li>
```

Comment:
Is this not needed anymore then? Is there a duplicateDefAdv() method that also needs removing then?

Reply:
the validation was removed in another story, this was missed out